### PR TITLE
[config] fix local file endings to match contents

### PR DESCRIPTION
**Pull request by gregnazario**
Several files are using "toml" as the file ending but are actually using
json or yaml.  Made it a little confusing surprise every time the files are opened!

### CODE REVIEW

File: config/src/config/secure_backend_config.rs [hunk 1]
```diff
@@ -104,7 +104,7 @@ impl Default for OnDiskStorageConfig {
     fn default() -> Self {
         Self {
             namespace: None,
-            path: PathBuf::from("secure_storage.toml"),
+            path: PathBuf::from("secure_storage.json"),
```
Comment (davidiw) [inline config/src/config/secure_backend_config.rs:107]: this is fine!

File: config/config-builder/src/main.rs [hunk 2]
```diff
@@ -9,8 +9,8 @@ use libra_network_address::NetworkAddress;
 use std::{convert::TryInto, fs, fs::File, io::Write, net::SocketAddr, path::PathBuf};
 use structopt::StructOpt;
 
-const KEY_MANAGER_CONFIG: &str = "key_manager.config.toml";
-const NODE_CONFIG: &str = "node.config.toml";
+const KEY_MANAGER_CONFIG: &str = "key_manager.config.yaml";
```
Comment (davidiw) [inline config/config-builder/src/main.rs:12]: we'll need to update the docker containers to load the right file.

**Comment by rexhoffman** [comment]
Recommend a codesearch in the libra/novifinancial org for the old file names -- should be easy to eyeball from there as a good last minute check.

**Comment by gregnazario** [comment]
> Recommend a codesearch in the libra/novifinancial orgs for the old file names -- should be easy to eyeball from there as a good last minute check.

I went ahead and did the searches.

Looks like the website and one tool use these files.  I could update them as well.

**Pull request closed without merging**